x/wasm/keeper/keeper.go

```diff
@@ -1081,6 +1215,18 @@ func (c BankCoinTransferrer) TransferCoins(parentCtx sdk.Context, fromAddr sdk.A
 	return nil
 }
 
+func (c BankCoinTransferrer) TransferCoinsToModule(ctx sdk.Context, fromAddr sdk.AccAddress, recipientModule string, amount sdk.Coins) error {
```

Comment:
nit, we may want to use the deferred send to module for this if we expect a lot of these action to occur (assuming this only happens on deposit rent, so maybe its infrequent enough as to not degrade concurrency, but if it becomes problematic we can make this a concurrent safe process)